### DISCUSSION TITLE
[질문] Hilt dependency가 없다고 나와요

### DISCUSSION THREAD

**Discussion by boogi-woogi** [TEAM-DAME/Dame-Android#43] [2. Q&A]
제가 project를 다시 클론받고 실행시켰는데 갑자기 아래와 같은 오류가 났습니다ㅠㅠ.
그리고 모듈을 표시하는 아이콘 모양도 그냥 모두 일반적인 폴더로 바뀌어 버렸습니다. 혹시 어떤게 문제일까요.
![image](https://user-images.githubusercontent.com/88770426/201647110-0a4f8233-726c-4491-876b-e2041661edda.png)
![image](https://user-images.githubusercontent.com/88770426/201647399-366b5226-c37f-4b9e-8e0b-2f5c246b5558.png)

**Comment by onseok** [comment]
`buildSrc` 모듈 내의 dependencies 등 파일들, 각 모듈 별로 build.gradle.kts 파일들을 건드리신게 아니라면 위와 같은 오류가 생기지 않을 것 같지만, 
의심 가는 부분은 안드로이드 스튜디오의 자바 JDK가 11로 설정되어 있지 않은지 다시 한번 확인해보시면 좋을 것 같습니다. 해당 프로젝트는 자바 11버전 기준으로 빌드됩니다.

<img width="952" alt="image" src="https://user-images.githubusercontent.com/76798309/201653181-1714e745-69e2-40db-8ef6-97769c02f5a5.png">

해당 방법으로도 정상적으로 빌드가 되지 않는다면 다시 말씀해주세요.

**Comment by boogi-woogi** [comment]
![image](https://user-images.githubusercontent.com/88770426/201653742-62b06eb6-4ad1-45b0-a3ea-5b228fc8175b.png)
jdk는 11로 설정이 되어있구 dependency를 따로 건드리지는 않았습니다. 그래도 혹시 제가 잘못 건드린게 있을 수 있으니 완전 처음부터 다시 클론받고 실행해보겠습니다!

**Reply by onseok** [reply]
`local.properties` 문제일수도 있겠군여
